server/conf.go

```diff
@@ -114,6 +114,21 @@ func ProcessConfigFile(configFile string, opts *Options) error {
 				return err
 			}
 			opts.AckSubsPoolSize = int(v.(int64))
+		case "ft_group", "ft_group_name":
```

Comment:
No, this was taken from NATS server code. It is basically any parameter name that I think would make it easier to use or remember. In some NATS Server you will even find some with typos, which is on purpose: Derek added the ones that he usually get wrong ;-)